src/components/FromSTIX/Statistics.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styles from "./from_stix.module.scss";
+import {useSelector} from "react-redux";
+import "./Statistics.scss";
+
+
+const Statistics = () => {
+    const mapping = useSelector((state) => state.fromStix.mapping);
+    const stixVersion = useSelector((state) => state.fromStix.stixFields);
+
+    function CalculateNumberOfCustomObjects(mapping){
+        var mylist = [];
```

Comment:
Please use `let` & `const` instead of `var` 
read more - https://dev.to/mindninjax/stop-using-var-for-declaring-variables-2p3a